rs/registry/canister/src/mutations/node_management/do_update_node_ipv4_config_directly.rs

```diff
@@ -80,15 +74,8 @@ impl Registry {
 
         // Ensure validity of IPv4 config (if it is present)
         if let Some(ipv4_config) = &payload.ipv4_config {
-            check_ipv4_config(
```

Comment:
Since you don't use `new` above, the checks that are now moved into the Ipv4Config new are never run. It is important to run them because this endpoint (`do_update_node_ipv4_config_directly`) can be called by any node provider.